docs/content/components/box.mdx

```diff
@@ -2,7 +2,49 @@
 title: Box
 ---
 
-# Box-Component
+# Box
+
```

Comment:
I would add here Description headline like in the other components and then link to your box primitive description